test/agents/test_chat_agent.py

```diff
@@ -423,19 +423,17 @@ def test_token_exceed_return():
     )
```

Comment:
Ummm. I don't think this unit test is meaningful. `step_token_exceed` will never be directly called by user.